apps/windows/package.json

```diff
@@ -8,7 +8,9 @@
     "lint": "eslint .",
     "windows": "react-native run-windows",
     "bundle": "fluentui-scripts metro --cli",
-    "e2etest": "rimraf reports/* && wdio"
+    "e2etest": "rimraf reports/* && wdio",
+    "report": "allure generate allure-results --clean",
+    "generate-report": "allure generate allure-results --clean && allure open"
```

Comment:
Do you need both of these?

Reply:
One is for CI and it just bundles the reports, the other is for local and bundles+opens report.